frontend/src/pages/home/home.tsx

```diff
@@ -71,11 +82,18 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
 
     render() {
         return (
-            <div className="home-root">
-                <Fade mounted={this.state.loading} childComponent={<Spinner/>}/>
-                <ArchiveSection />
-                {this.renderMessageCardSection()}
-            </div>
+            <section id='anchor'>
+                <div className="home-root">
+                    <Fade mounted={this.state.loading} childComponent={<Spinner/>}/>
+                    <ArchiveSection />
+                    <div className="justify-center padding-top">
+                        <div className="justify-align-center">
```

Comment:
Maybe instead of aligning to center, just make it one "column". There shouldn't be more than 2-4 announcements anyways. Also, at this resolution, it kinda looks bad
![localhost_3000_home](https://user-images.githubusercontent.com/6146988/95936940-881ab900-0d8b-11eb-97af-b4e4a5f3da1c.png)

Reply:
I changed the style so the cards stack